extensions/wikia/AdEngine/js/pageFairDetection.js

```diff
@@ -60,6 +71,7 @@ define('ext.wikia.adEngine.pageFairDetection', [
 	}
 
 	return {
+		canBeInitialized: canBeInitialized,
```

Comment:
I wanted to validate context inside initDetection method and additionally allow to check it outside

1. probably I will do it in next change 
2. nope, SourcePoint has check-method inside initDetection